src/main/resources/msmarco_v1/msmarco-v1-passage.yaml

```diff
@@ -0,0 +1,300 @@
+conditions:
```

Comment:
We already have `src/main/java/io/anserini/reproduce/msmarco-v1-passage.yaml` right? Can't we just use that?

Reply:
I believe all the files in `src/main/resources/` can be read within a fatjar. To read files from `src/main/java/io/anserini/reproduce/msmarco-v1-passage.yaml` we would need to specify that directory as a resource in `pom.xml`.